mease_lab_to_nwb/convert_syntalos/syntalosimageinterface.py

```diff
@@ -23,30 +22,24 @@ def get_source_schema(cls):
         )
 
     def run_conversion(self, nwbfile: NWBFile, metadata: dict):
-        """
-        Primary conversion function for the custom Syntalos image interface.
-
-        Parameters
-        ----------
-        nwbfile : NWBFile
-        metadata_dict : dict
-        stub_test : bool, optional
-            If true, truncates all data to a small size for fast testing. The default is False.
-        """
```

Comment:
why did you delete this?

Reply:
I realized the docstrings for the abstract functions are inherited  from the BaseDataInterfaces, so I'm getting into the habit of not duplicating them every time unless there are specific overrides to add extra arguments.

Reply:
oh, I see this feature of python was added in 3.5. Huh, I didn't know that! OK, carry on deleting, looks good to me.

Reply:
Yeah I had no clue either until I tried it - realizing now that we can cut down a lot of downstream code text by centralizing all of it on the nwb-conversion-tools side, which I'll work on polishing over the next week or two